src/core/public/chrome/ui/header/header.tsx

```diff
@@ -659,15 +668,9 @@ export function Header({
     return useApplicationHeader ? renderApplicationHeader() : renderPageHeader();
   };
 
-  // Get the banner plugin configuration
-  const bannerPluginConfig = injectedMetadata
-    ?.getPlugins()
-    ?.find((plugin: { id: string }) => plugin.id === 'banner')?.config;
-  const isBannerEnabled = bannerPluginConfig?.enabled === true;
-
   return (
     <>
-      {isBannerEnabled && <div id="pluginGlobalBanner" />}
+      {isBannerEnabled && !useUpdatedHeader && <div id="pluginGlobalBanner" />}
```

Comment:
Why not always just have a space to mount global banners vs making it specific to this plugin?